apps/filebrowser/src/filebrowser/views.py

```diff
@@ -1458,10 +1458,11 @@ def _create_response(request, _fs, result="success", data="Success"):
       'success': True,
       'uuid': _fs.qquuid,
       'status': 0,
-      'data': data
+      'data': data,
+      'task_id': _fs.qquuid
   }
 
-def perform_upload(request, *args, **kwargs):
+def perform_upload_task(request, *args, **kwargs):
```

Comment:
Why rename the function? It seems that it still supports the non task server method.